package.json

```diff
@@ -52,15 +52,15 @@
     "zopflipng"
   ],
   "dependencies": {
-    "bin-build": "^2.2.0",
-    "bin-wrapper": "^3.0.0",
+    "bin-build": "^3.0.0",
+    "bin-wrapper": "^4.0.0",
     "logalot": "^2.0.0"
   },
   "devDependencies": {
     "ava": "*",
```

Comment:
Can you use a version range for ava and xo? Things break frequently due to this.